src/test/java/com/acrolinx/sidebar/swing/AcrolinxSidebarSwingTest.java

```diff
@@ -0,0 +1,42 @@
+/* Copyright (c) 2017-present Acrolinx GmbH */
+
+package com.acrolinx.sidebar.swing;
+
+import java.awt.event.KeyEvent;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import com.acrolinx.sidebar.AcrolinxIntegration;
+
+class AcrolinxSidebarSwingTest
+{
+
+    private static void testPasteCommand(int key, int modifier, boolean consume)
```

Comment:
Having a boolean argument is a code smell, a better solution is usually to have two separate methods (without a boolean argument) for the two different cases with clear/different method names.

Reply:
Okay